assets/js/modules/analytics-4/datastore/audiences.test.js

```diff
@@ -159,19 +157,75 @@ describe( 'modules/analytics-4 audiences', () => {
 						},
 					},
 				} );
+			} );
+		} );
 
-				expect( store.getState().audiences.length ).toBe( 1 );
-				expect( store.getState().audiences[ 0 ] ).toEqual(
-					audiencesFixture[ 2 ]
-				);
+		describe( 'getAvailableAudiences', () => {
+			const availableAudiences = [
+				{
+					name: 'properties/123456789/audiences/0987654321',
+					displayName: 'All visitors',
+					description: 'All users',
+					audienceType: 'DEFAULT_AUDIENCE',
+					audienceSlug: 'all-users',
+				},
+			];
+
+			it( 'should not sync cached audiences when the availableAudiences setting is not null', () => {
+				fetchMock.postOnce( syncAvailableAudiencesEndpoint, {
+					body: availableAudiences,
+					status: 200,
+				} );
```

Comment:
Using fetchMock here shouldn't be necessary since the expectation is for it not to be fetched. @ankitrox

Reply:
Thanks here too, @aaemnnosttv, I am removing this call in my current PR too.